op-batcher/throttling.md

```diff
@@ -15,10 +15,22 @@ The throttling system prevents these issues by instructing sequencers to limit b
 
 ## Throttling Controller Types
 
-The batcher supports four throttling strategies, each with different response characteristics:
+The batcher supports four throttling strategies, each with different response characteristics. The strategies can be understood in this
+diagram:
+![Throttling Strategies](./images/throttling.png)
+
+* Each strategy responds to the `unsafe_da_bytes` metric and corresponding thresholds `throttle.unsafe-da-bytes-lower/upper-threshold`, and results a throttling "intensity" between 0 and 1.
+
+* This intensity is then mapped to a maximum tx size and maximum block size to control the `miner_setMaxDASize(maxTxSize, maxBlockSize)` API calls made to block builders, depending on the configuration variables shown in the diagram above.
+
+* When the throttling intensity is zero (the `unsafe_da_bytes` is less than `unsafe-da-bytes-lower-threshold`), blocks will continue to be limited at `throttle.block-size-upper-limit`, whereas transactions are not throttled at all (by using `maxTxSize=0`).
+
+> NOTE
+> Be aware that using `0` for either
+> `throttle.block-size-lower-limit` and `throttle.tx-size-lower-limit`
+> will be interpreted by block builders as "no limit"
 
 ### Step Controller (Default)
```

Comment:
Maybe we should put a warning that the step controller should not be used with default values? Because it would too soon throttle too hard?